src/replica/mutation.h

```diff
@@ -155,7 +160,15 @@ class mutation : public ref_counter
         {
             unsigned int _not_logged : 1;
             unsigned int _left_secondary_ack_count : 15;
-            unsigned int _left_potential_secondary_ack_count : 16;
+            unsigned int _left_potential_secondary_ack_count : 14;
+            // Used for partition split
+            // _wait_child = 1 : child prepare mutation synchronously, its parent should wait for
+            // child ack
+            unsigned int _wait_child : 1;
```

Comment:
You can use bool here, I've tested on c++98,c++11,c++14, with `-wall` flag on.

```
// Example program
#include <iostream>
#include <string>

struct CompactStruct {
    unsigned int _not_logged : 1;
    unsigned int _left_secondary_ack_count : 15;
    unsigned int _left_potential_secondary_ack_count : 16;
    // Used for partition split
    // _wait_child = 1 : child prepare mutation synchronously, its parent should wait for
    // child ack
    bool _wait_child : 1;
    // Used for partition split
    // when prepare failed when child prepare mutation synchronously, secondary may try to
    // ack to primary twice, we use _is_error_acked to restrict only ack once
    bool _is_error_acked : 1;
};

int main()
{
    CompactStruct t;
    t._is_error_acked = 1;
    std::cout << t._wait_child << std::endl;
    return 0;
}
```

Reply:
Done